### ISSUE TITLE
Visual glitch with bigger font

### ISSUE THREAD

**Issue by piotr-yuxuan** [emacs-tree-sitter/ts-fold#122]
Hello, thanks a lot for this great package 😊

When setting the default face size at 180 instead of the usual 100 the fringe indicators are rendered this way:

![image](https://github.com/emacs-tree-sitter/ts-fold/assets/3686502/9a0edce3-3fc0-460d-bf05-71a0dfc7e5ef)

and I need to manually overload these values:

```` elisp
'ts-fold-indicators-fr-minus-tail
'ts-fold-indicators-fr-center
'ts-fold-indicators-fr-end-left
'ts-fold-indicators-fr-end-right
````
with some more (balanced) lines
``` elisp
  "........" "........" "........" "........" "........"
  "...XX..." "...XX..." "...XX..." "...XX..." "...XX..."
```

to get a visual fix:
![image](https://github.com/emacs-tree-sitter/ts-fold/assets/3686502/9d06618b-eed4-44b9-af11-160c53f6f940)

I'm wondering what a more flexible fix would look like.

**Comment by jcs090218** [comment]
Yeah, this could be an issue since I don't know if there is a flexible way to define these values.🤔

**Comment by piotr-yuxuan** [comment]
Yeah, it's likely to incur at least a partial rewrite :/ But anyway, thanks a lot for this nice package :-)

**Comment by piotr-yuxuan** [comment]
I resolved it by using https://github.com/blahgeek/emacs-fringe-scale:
``` elisp
(set-fringe-mode 16)
(fringe-scale-setup)
```
